collectors/python.d.plugin/python_modules/bases/FrameworkServices/UrlService.py

```diff
@@ -88,13 +88,13 @@ def _build_manager(self, **header_kw):
             self.error('build_manager() error:', str(error))
             return None
 
-    def _get_raw_data(self, url=None, manager=None):
+    def _get_raw_data(self, url=None, manager=None, **kwargs):
```

Comment:
Could you elaborate a bit on the necessity of the variable arguments on the core library item?

Reply:
This is needed to pass arbitrary named arguments to urrlib3.PoolMannager().request method

> Signature: m.request(method, url, fields=None, headers=None, **urlopen_kw)

https://github.com/netdata/netdata/blob/d242e02452ff309334b05a17d6e1061ea3caeac8/collectors/python.d.plugin/python_modules/bases/FrameworkServices/UrlService.py#L119-L127

And it it already added in #5893

Reply:
i think @andvgal better to rebase and undo these commits otherwise it is confusing

Reply:
Done, rebased